### PR TITLE
Increase full fetch window for Nitro TV content to 7 days ahead

### PR DESCRIPTION
**Pull request by oli-hall**

### CODE REVIEW

File: src/main/java/org/atlasapi/remotesite/bbc/nitro/LocalOrRemoteNitroFetcher.java [hunk 1]
```diff
@@ -63,7 +63,7 @@ public boolean apply(Item input) {
                         final Interval fetchForBroadcastsWithin = 
                                 MediaType.AUDIO.equals(input.getMediaType()) 
                                     ? broadcastInterval(today.minusDays(5), today.plusDays(1)) 
-                                    : broadcastInterval(today.minusDays(3), today.plusDays(3));
+                                    : broadcastInterval(today.minusDays(3), today.plusDays(7));
```
Comment (tfm) [inline src/main/java/org/atlasapi/remotesite/bbc/nitro/LocalOrRemoteNitroFetcher.java]: Would be more inclined to put in an explicit `or broadcast is in the future` condition. Then the intention is really clear.

Comment (oli-hall) [inline src/main/java/org/atlasapi/remotesite/bbc/nitro/LocalOrRemoteNitroFetcher.java]: Not sure what you mean... wrap the entire predicate? surely we only care about TV content in the future?

Comment (tfm) [inline src/main/java/org/atlasapi/remotesite/bbc/nitro/LocalOrRemoteNitroFetcher.java]: something like 

```
public boolean apply(org.atlasapi.media.entity.Broadcast input) {
                                return 
input.getTransmissionTime().isAfterNow() ||
fetchForBroadcastsWithin.contains(input.getTransmissionTime());
                            }});
```

Comment (oli-hall) [inline src/main/java/org/atlasapi/remotesite/bbc/nitro/LocalOrRemoteNitroFetcher.java]: OK, will go with that. I'm sensing I've not quite got the full context around this, but will go with your suggestion for now.

**Pull request merged**